### DISCUSSION TITLE
Backup/Migration without Docker?

### DISCUSSION THREAD

**Discussion by PlaysWithTrons** [teslamate-org/teslamate#3375] [Q&A]
I've made a horrible mistake and now am trying to recover.

I have been running teslamate very successfully on a Synology NAS for the last two years. Then, the NAS crashed. I currently have access to the files on the NAS, but it is in READ ONLY mode, without docker running. I do not have a backup (I told you I made a horrible mistake).

I now have a new NAS with teslamate installed (very easy.. Is there any way to migrate/backup/import my old data into the new install? I am willing to lose the most recent new data if there is a way to recover my lost two years of data. 

Lesson learned.

**Comment by DrMichael** [comment]
If you could somehow get the volume of the database service, you could reuse that. (Cannot tell hwo though)
